src/types/RoleToEmojiData.ts

```diff
@@ -36,4 +36,12 @@ export interface RoleToEmojiData {
    * @memberof RoleToEmojiData
    */
   removeOnReact?: boolean;
+
+  /**
+   * List of required roles to be able to react to this role.
+   *
+   * @type {RoleResolvable[]}
+   * @memberof RoleToEmojiData
+   */
+  dependencies?: RoleResolvable[];
```

Comment:
I am not a fan of the "dependencies" word. I would prefer a more user-friendly naming, such as `requiredRoles` for example ? You yourself name it like that in the JSDocs right above.